src/uct/tcp/tcp.h

```diff
@@ -350,6 +349,7 @@ struct uct_tcp_ep {
         ucs_conn_match_elem_t     elem;         /* Connection matching element, used by EPs
                                                  * created with CONNECT_TO_IFACE method */
     };
+    struct sockaddr_storage       peer_addr;    /* Remote iface addr */
```

Comment:
the endpoint size should be according to actual address size, not the max possible size